samcli/lib/cli_validation/remote_invoke_options_validations.py

```diff
@@ -50,7 +50,7 @@ def wrapped(*args, **kwargs):
 
         # if no event nor event_file arguments are given, read from stdin
         if not event and not event_file:
-            LOG.debug("Neither --event nor --event-file options have been provided, reading from stdin")
+            LOG.info("Neither --event nor --event-file options have been provided, reading from stdin")
             kwargs["event_file"] = cast(TextIOWrapper, sys.stdin)
```

Comment:
I thought we decided to set this to empty event to align with `sam local invoke`?

Reply:
Yes those changes are in the UX PR. Once that gets merged, I will resolve the conflicts in this PR.